### PR TITLE
chore(reports): remove HTML report generation

### PR DESCRIPTION
**Pull request by andrewazores**
Depends on https://github.com/cryostatio/cryostat-core/pull/261

Remove HTML generation API and do some cleanup.

### CODE REVIEW

**Comment by andrewazores** [comment]
https://github.com/cryostatio/cryostat-core/pull/261#issuecomment-1737385721

**Comment by andrewazores** [comment]
Required `-core` is released.

File: src/main/java/io/cryostat/reports/ReportResource.java [hunk 1]
```diff
@@ -135,56 +96,39 @@ public String getEval(RoutingContext ctx, @BeanParam RecordingFormData form)
             throws IOException {
         FileUpload upload = form.file;
 
-        Triple<java.nio.file.Path, ReportGenerationEvent, Pair<Long, Long>> tripleHelper =
-                reportHelper(upload);
+        Pair<java.nio.file.Path, Pair<Long, Long>> tripleHelper = reportHelper(upload);
```
Comment (mwangggg) [inline src/main/java/io/cryostat/reports/ReportResource.java:99]: rename to pairHelper?

**Review by aali309 — COMMENTED** [review]
Had this at first `cryostat-reports: Detected Maven Version (3.8.1)  is not supported, it must be in [3.8.2,).`
had to update `/cryostat-reports/.mvn/wrapper/maven-wrapper.properties` to the suggested `3.8.2`
Other than that, looks good to me.

**Comment by andrewazores** [comment]
Ah interesting, I haven't run `./mvnw quarkus:dev` in quite some time so I hadn't noticed this. I'll open a separate PR.

**Pull request merged**